app/controllers/idv/in_person/address_controller.rb

```diff
@@ -5,14 +5,39 @@ class AddressController < ApplicationController
 
       before_action :render_404_if_in_person_residential_address_controller_enabled_not_set
       before_action :confirm_in_person_state_id_step_complete
-      before_action :confirm_in_person_address_step_needed
+      before_action :confirm_in_person_address_step_needed, only: :show
+      before_action :confirm_ssn_step_needed
+
+      attr_accessor :error_message
```

Comment:
TIL that this was available in controllers.

That said, it's not obvious to me where this is used?

Reply:
thanks for catching this -> it's a holdover from something i was testing, it's actually not needed